contrib/hashicorp/consul/benchmark_test.go

```diff
@@ -0,0 +1,75 @@
+package consul
+
+import (
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	consul "github.com/hashicorp/consul/api"
+)
+
+func BenchmarkKV(b *testing.B) {
+	key := "test.key"
+	pair := &consul.KVPair{Key: key, Value: []byte("test_value")}
+	testCases := []struct {
```

Comment:
This could simply be a `map[string]func(*consul.KV) error` where the map key is the test name.

Reply:
Done